### DISCUSSION TITLE
update query Issue

### DISCUSSION THREAD

**Discussion by amantulsyan35** [prisma/prisma#13852] [Q&A]
Hii, Is there any way of deleting a child table row in an update query ?

**Comment by nurul3101** [comment]
Hey @amantulsyan35 👋 
Could you please explain your use case?
Which database are you using?

**Reply by vladi-stevanovic** [reply]
👋 Hello @amantulsyan35  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
